lib/src/cli/command/bump_command.dart

```diff
@@ -1,27 +1,49 @@
+import 'package:args/command_runner.dart';
 import 'package:cider/src/project.dart';
+import 'package:cider/src/cli/command/bump_sub_command.dart';
 import 'package:cider/src/cli/command/cider_command.dart';
 import 'package:version_manipulation/mutations.dart';
 
+enum BumpType {
+  breaking,
+  major,
+  minor,
+  patch,
+  build,
+  pre,
+  release,
+}
+
 class BumpCommand extends CiderCommand {
   BumpCommand(super.printer) {
-    mutations.keys.forEach(argParser.addCommand);
-    argParser
-      ..addFlag('keep-build', help: 'Keep the existing build')
-      ..addFlag('bump-build', help: 'Also bump the build')
-      ..addOption('build',
-          help: 'Sets the build to the given value', defaultsTo: '')
-      ..addOption('pre',
-          help: 'Sets the pre-release to the given value', defaultsTo: '');
+    for (final type in BumpType.values) {
+      addSubcommand(BumpSubCommand(
+        type.name,
+        description: _subcommandDescriptions[type]!,
+        mutation: _mutations[type]!,
+        printer: printer,
+      ));
+    }
   }
 
-  static const mutations = <String, VersionMutation>{
-    'breaking': BumpBreaking(),
-    'build': BumpBuild(),
-    'major': BumpMajor(),
-    'minor': BumpMinor(),
-    'patch': BumpPatch(),
-    'pre': BumpPreRelease(),
-    'release': Release(),
+  static const _mutations = <BumpType, VersionMutation>{
```

Comment:
Would it be cleaner to make these properties of the `BumpType` enum?

Reply:
I didn't even know you could do that with enums in Dart. Anyway, that does make sense.